src/main/modules/properties-volume/index.ts

```diff
@@ -12,6 +12,10 @@ export class PropertiesVolume {
       this.setSecret('secrets.idam-idam.AppInsightsInstrumentationKey', 'appInsights.instrumentationKey');
       this.setSecret('secrets.idam-idam.launchdarkly-sdk-key', 'launchdarkly.sdkKey');
       this.setSecret('secrets.idam-idam.idam-user-dashboard-client-secret', 'services.idam.clientSecret');
+      this.setSecret('secrets.idam-idam.redis-hostname', 'session.redis.host');
+      this.setSecret('secrets.idam-idam.redis-port', 'session.redis.port');
+      this.setSecret('secrets.idam-idam.redis-key', 'session.redis.key');
+      this.setSecret('secrets.idam-idam.redis-key', 'session.secret');
```

Comment:
You are adding the redis-key as secret here....Is that intentional to use the key as secret?

Reply:
Yep, seems common place (at least in HMCTS) to use it as the session secret